documents/policies/internal/Working Group Formation.md

```diff
@@ -1,15 +1,15 @@
 # WCA Working Group Formation Policy
 
-### Version 1.0 {.version}
+### Version 1.1 {.version}
 
 ## Purpose
 The purpose of this policy is to provide guidelines for the formation and operation of working groups within the World Cube Association (WCA) in order to facilitate collaboration, innovation, and progress in specific focus areas.
 
 ## Policy
 1. Definition of a Working Group
-   1. A working group is a temporary team of WCA Staff and/or community members established to address a specific project or area of interest within the organization. Working groups are tasked with generating ideas, providing recommendations, and executing initiatives related to their assigned focus area
+   1. A working group is a temporary team of WCA Volunteers and/or community members established to address a specific project or area of interest within the organization. Working groups are tasked with generating ideas, providing recommendations, and executing initiatives related to their assigned focus area
```

Comment:
re: "community members", "Member of the WCA Community" is now used in the CoC above, but is not currently defined in the Definitions Motion. A consistent approach on this term would be desirable.